llm/codellama/complete.py

```diff
@@ -0,0 +1,27 @@
+import openai
+import sky
+from sky import serve
+
+# service_records = sky.serve.status('code-llama')
+# endpoint = serve.get_endpoint(service_records[0])
+
+ip = sky.status('test-endpoint')[0]['handle'].external_ips()[0]
+endpoint = f'{ip}:8000'
```

Comment:
Note for feature - we should probably add a nice programattic way to get endpoints : )

Reply:
Yep, just filed an issue for this #3053